### PR TITLE
camera_itof.cpp: mixed modes check for any version different from 0

### PR DESCRIPTION
**Pull request by rbudai98**

### CODE REVIEW

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 1]
```diff
@@ -263,41 +263,38 @@ aditof::Status CameraItof::initialize() {
                     m_modesVersion = 0;
                 }
             }
+        }
+        // Check weather new mixed modes are present, in case no, switch back to simple new modes
+        if (m_modesVersion == 2) //new mixed modes table is active
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:268]: I don't understand the comment. Why you say new mixed modes table is active?

File: sdk/src/cameras/itof-camera/camera_itof.cpp [hunk 2]
```diff
@@ -263,41 +263,38 @@ aditof::Status CameraItof::initialize() {
...
+        {
+            int modeToTest = 5; // We are looking at width and height for mode 0
```
Comment (dNechita) [inline sdk/src/cameras/itof-camera/camera_itof.cpp:270]: comment should say mode 5 not 0

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 1 above)
Comment (rbudai98) [reply sdk/src/cameras/itof-camera/camera_itof.cpp:268]: The comment remained there from previous implementations. Will remove it

File: sdk/src/cameras/itof-camera/camera_itof.cpp (hunk 2 above)
Comment (rbudai98) [reply sdk/src/cameras/itof-camera/camera_itof.cpp:270]: will change it to 5

**Review by dNechita — APPROVED** [review]

**Pull request merged**